### DISCUSSION TITLE
direactus_collections - translations have wrong data type

### DISCUSSION THREAD

**Discussion by anlexN** [directus/directus#15632] [Archive]
![image](https://user-images.githubusercontent.com/8686739/190892854-2599a4b4-b797-46d1-8339-90c8cbcfdbda.png)

![image](https://user-images.githubusercontent.com/8686739/190893665-3b929ce5-4497-4242-a084-5b09eb799d55.png)

`item_duplication_fields` is too.

**Comment by rijkvanzanten** [comment]
Translations are a json array of individual objects, so it's correct

**Reply by anlexN** [reply]
so why use two different words?

**Reply by br41nslug** [reply]
Because the docs are explaining the type of data you're retrieving from the API while the specific database column type may be different for instance `json` or `string` depending on your database engine.

**Reply by anlexN** [reply]
so it's very confusing.

**Reply by br41nslug** [reply]
Having an abstraction layer supporting multiple database vendors with they're own features and edge-cases can get confusing indeed 😄
